tables/Makefile.am

```diff
@@ -321,6 +321,7 @@ table_files = \
 	pu-in-g1.utb \
 	ro.ctb \
 	ro.tbl \
+	ro-g0.ctb \
```

Comment:
The name should be ro-g0._utb_. The ".ctb" extension is intended for table for contracted braille.

Reply:
Done